.travis.yml

```diff
@@ -1,7 +1,7 @@
 rvm:
-  - 1.9.3
   - 2.0.0
-  - 2.1.0
+  - 2.1.3
```

Comment:
2.1.7 should work on travis ... afaik the only downside is that needs more
time because it is compiled and not pre-installed

On Mon, Dec 21, 2015 at 8:00 AM, Benjamin Quorning <notifications@github.com
> wrote:

> In .travis.yml
> <https://github.com/zendesk/active_record_host_pool/pull/12#discussion_r48159163>
> :
>
> >    - 2.0.0
> > -  - 2.1.0
> > +  - 2.1.3
>
> Travis goes only up to 2.1.5, but I had some trouble bundling. Will retry.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/active_record_host_pool/pull/12/files#r48159163>
> .
>